src/SonarScanner.MSBuild.PreProcessor/SonarScanner.MSBuild.PreProcessor.csproj

```diff
@@ -18,7 +18,9 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Mono.Posix.NETStandard" Version="1.0.0" />
```

Comment:
It looks like [this mono.posix the successor of Mono.Posix.NETStandard](https://github.com/xamarin/XamarinComponents/issues/1102#issuecomment-850962005)
https://github.com/mono/mono.posix
https://www.nuget.org/packages/Mono.Posix/7.1.0-final.1.21458.1

Reply:
The project with the FilePermission class from https://github.com/SonarSource/sonar-scanner-msbuild/pull/2036/files#r1673558952 is also using the new package:
https://sourcegraph.com/github.com/Jackett/Jackett/-/blob/src/Jackett.Server/Jackett.Server.csproj?L59